homeassistant/components/channels/icons.json

```diff
@@ -0,0 +1,7 @@
+{
+  "services": {
+    "seek_forward": "mdi:fast-forward",
+    "seek_backward": "mdi:skip-backward",
```

Comment:
Should we use mdi:skip-forward for seek_forward (or mdi:rewind for seek_backward) so they're more consistent?